### PR TITLE
abi: user: make userspce ABI usage more obvious

### PR DESCRIPTION
**Pull request by lgirdwood**
Clear the fog around usage of ABI header for userspace applications. Also make the sof-eqctl generic as there is nothing that tightly coupes it to equalizers (except strings).

### CODE REVIEW

File: tools/ctl/ctl.c [hunk 1]
```diff
@@ -9,29 +9,24 @@
 #include <stdint.h>
 #include <errno.h>
 #include <alsa/asoundlib.h>
-
-#define SOF_CTRL_CMD_BINARY 3 /* TODO: From uapi ipc */
```
Comment (singalsu) [inline tools/ctl/ctl.c]: Good, thanks for cleaning up this!

**Comment by xiulipan** [comment]
@lgirdwood 
I can see some some internal error in CI as we have changed the name of `sof-eqctl ` => `sof-ctl` as well as 
the folder. I will temporarily remove the storage for eqctl or ctl until the naming is stable.

**Comment by lgirdwood** [comment]
@zrombel does this sof-ctl binary name change impact internal CI like @xiulipan is reporting on Jenkins ? It looks like it's been pending a while ?

**Comment by zrombel** [comment]
@lgirdwood this change doesn't affect FW CI. It's good to go.

**Pull request merged**